charter.md

```diff
@@ -63,6 +63,30 @@ are able to proceed with a meeting.
 * Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
 * Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
 
+## Elections
+
+Every year, the Steering Committee holds an election for open seats with SIG 
+Contributor Experience owning the process. Our [election policy document](https://git.k8s.io/steering/elections.md) 
+covers the details for how this works.
+### Vacancies and Removal
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+#### Removal
+
+A Steering Committee member may be removed by a two-thirds vote of the other 
```

Comment:
Quickly running the math on this, the outcome with seven members appears subpar: 4/6 is the same as simple majority vote.  The voting thresholds were developed with the size of the committee in mind and modeling this as a full committee vote where the removal target also gets to vote produces a better outcome where 5/7 members of the committee must agree, which matches the threshold for existing votes.

Reply:
General practice has been to round up - this would be 5/7, from the charter on quorum:

```
Quorum
Steering committee members are generally expected to attend every meeting. We use the following guidelines to determine whether we have reached quorum and are able to proceed with a meeting.

Round quorum up if total number of members is odd
Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
```

Reply:
> General practice has been to round up - this would be 5/7, from the charter on quorum:

The exact phrasing in this PR currently says 

> a two-thirds vote of the other Steering Committee members

So if there are 7 members, and 1 is being targeted for removal, the base number would become 6.  If the PR was rephrased to be 

> a two-thirds vote of the Steering Committee members

then it would be 5/7.  Can we get it rephrased to make this clear?

Reply:
I don't think it should be codified specifically as an x/y, since then if for any reason the steering committee size would change over time, this number would need to be modified...  Maybe clarify a bit more that the 2/3 implies rounding up the number of votes needed when the number of members is not a multiple of 3?

Reply:
> I don't think it should be codified specifically as an x/y, since then if for any reason the steering committee size would change over time,

I agree.  I think if the current intent is to be 5/7 instead of 4/7, that can be achieved by phrasing the requirement as 

> a two-thirds vote of the Steering Committee members

As currently stated, the threshold is 4/7 because the seventh was excluded from the vote.

Reply:
agree that we can clear this up, thinking, will come back with a change and ping y'all here.